zeppelin-interpreter/pom.xml

```diff
@@ -241,6 +241,42 @@
           <dateFormat>yyyy-MM-dd HH:mm:ss</dateFormat>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>3.0.0</version>
+        <executions>
+          <execution>
+            <!-- This execution runs during the compile phase to generate Java files from Thrift files -->
```

Comment:
Thrift is not installed in the CI pipeline, so these steps can only be executed using profiles similar to hbase-thrift.